docs/elections.rst

```diff
@@ -20,6 +20,14 @@ elections.local
    :undoc-members:
    :show-inheritance:
 
+elections.city\_of\_london\_local
+----------------------
+
+.. automodule:: uk_election_timetables.elections.city_of_london_local
+   :members:
+   :undoc-members:
+   :show-inheritance:
+
```

Comment:
This looks like it is probably right, but I haven't tested it.
I can't work out how to build the docs site locally for this project and I can't find any docs on it. Normally I would expect a site that uses sphinx for docs to have a Makefile with a command to build the docs, but the cupboard is bare. Any advice?

Side note: Looks like your docs deploy is broken anyway [![Documentation Status](https://readthedocs.org/projects/uk-election-timetables/badge/?version=latest)](https://uk-election-timetables.readthedocs.io/en/latest/overview.html?badge=latest)

Reply:
Looks like I tried to work on this earlier this year:

https://github.com/DemocracyClub/uk-election-timetables/pull/40/files

I think I ran `sphinx-build -M html docs htmldocs`, but I'm unsure. 

My memory is that this was done by magic with read the docs. @mrwilson controls the account and I think I asked them to make me an admin...but this was all pre general election, so who knows. 

Read the docs changed to requiring a config file checked in to the repo, hence the PR above, but it's  not going to build until we press some button in the read the docs admin, that I can't do